evalml/tests/component_tests/test_decision_tree_classifier.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier as SKDecisionTreeClassifier
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines import DecisionTreeClassifier
+from evalml.problem_types import ProblemTypes
+
+
+def test_model_family():
+    assert DecisionTreeClassifier.model_family == ModelFamily.DECISION_TREE
+
+
+def test_problem_types():
+    assert ProblemTypes.BINARY in DecisionTreeClassifier.supported_problem_types
+    assert ProblemTypes.MULTICLASS in DecisionTreeClassifier.supported_problem_types
+    assert len(DecisionTreeClassifier.supported_problem_types) == 2
+
+
+def test_fit_predict_binary(X_y_binary):
```

Comment:
According to the documentation, "DecisionTreeClassifier is capable of both binary (where the labels are [-1, 1]) classification and multiclass (where the labels are [0, …, K-1]) classification."

Might be useful to check that these results for the binary test (where labels are [0, 1]) are equal to what the results would be for labels [-1, 1]

Reply:
This is a good suggestion! I think we currently limit what we allow for targets to [0, 1] for binary classification and [0, …, K-1] for multiclass though, so we don't necessarily need to guarantee what happens when the user breaks that. (Best docs I can think of to show is this: https://evalml.featurelabs.com/en/latest/user_guide/components.html#Expectations-for-Custom-Classification-Components)

Reply:
Gotcha, sounds good. Thanks!